backend/tests/unit/core/schema_manager/test_manager_schema.py

```diff
@@ -1226,7 +1233,6 @@ async def test_validate_display_labels_error(schema_all_in_one, display_labels,
         ["my_generic_name__value", "mybool__value"],
         ["my_generic_name__value"],
         ["primary_tag__name__value"],
-        ["status__name__value", "mybool__value"],
```

Comment:
did this one start to fail? it seems like it should still be allowed

Reply:
As discussed, `order_by` should support optional cardinality-one relationship